internal/cmd/util_windows.go

```diff
@@ -17,6 +17,9 @@ var defaultInterpreters = map[string]*chezmoi.Interpreter{
 	"cmd": {},
 	"com": {},
 	"exe": {},
+	"nu": {
+		Command: "nu",
```

Comment:
Should this be `nu` or `nushell`?

Reply:
It should be `nu`, I've tested it locally and it runs the scripts fine.